### ISSUE TITLE
Experimental features and related news

### ISSUE THREAD

**Issue by FMorschel** [dart-lang/site-www#5370]
### What information needs to be added?

I expected the site also to give the programmers a little insight into what the Dart team is working on. For example the `extension types` (https://github.com/dart-lang/language/issues/2727) and some other experimental flags we could try out. Probably linked to the issues where they are being discussed. Maybe also link to the Medium posts about new releases and similar things.

I miss a way to be in the know about what you guys are thinking and be able to give my suggestions while the features are being worked on.

### Where should this new content appear?

Probably a new tab.

### I would like to fix this problem.

- [ ] I will try and fix this problem on dart.dev.

**Comment by parlough** [comment]
This is something I plan to work on, but I'm not yet completely sure the best way to do so. I'm not saying it's possible, but would something like https://chromestatus.com/roadmap but for Dart be helpful?

**Comment by FMorschel** [comment]
Yes! That would be really helpful. I even believe that folks who are willing to start supporting the Dart project with their commits would be more inclined to do so if these are available in an easy way for them to see on the official website.

Probably make them aware of what is being worked on, and the general idea of the community for the future of the language.

I really like the way the example you gave is visually, but I would also suggest linking Dart-related Medium posts on the top, before the version differentiations and maybe a link to the Dart UX surveys like I suggested here: https://github.com/flutter/uxr/issues/115

**Comment by FMorschel** [comment]
As I mentioned on https://github.com/flutter/website/issues/9848, there could be a link to that "new things" page or whatever when we do a `flutter upgrade` or something similar as well.

**Comment by mit-mit** [comment]
For the language, we have this tracker: https://github.com/dart-lang/language/projects/1

**Comment by FMorschel** [comment]
I see, that's a good place to look for this. My issue is just about making this link more accessible and easy to find for new developers who want to know what's going on.

**Comment by FMorschel** [comment]
Also, another thought I had. The release posts on Medium and related things could be listed on this page so people know where to look for differences in versions.

Like "When was that feature released?". Possibly linking to the official docs for those new stuff.

**Comment by FMorschel** [comment]
I found https://dartcode.org/releases/ today. This is a great example of what I'm asking. A simple link where I can easily see the changes for the SDK. Of course, there could be a link to the language project and related future features.

My point here is that I find it a bit odd that I have a way to follow Flutter releases (sync with Dart) on Discord, but for an actual description of what is new I should go to the Medium post and if I want to know what is being worked on I need to come back over here on GitHub to see the language project. 

I know that all of those can be seen here on GitHub but I don't think this is the right place to show them to the new user (or anyone) at least for now, IMO, this was not meant to be the place to look for explanations and updates.
